docs/usage/private-container-reg.md

```diff
@@ -0,0 +1,5 @@
+# Private container registries
+
+To access a private container registry, follow the steps on [this page](http://kubernetes.io/docs/user-guide/images/).
+
+We recommend you use ImagePullSecrets, but if you would like to configure access on the minishift VM you can place the `.dockercfg` in the `/home/docker` directory or the `config.json` in the `/home/docker/.docker` directory.
```

Comment:
Should this be `ImagePullSecrets` or  `imagePullSecrets` or stay as is? As per this link: http://kubernetes.io/docs/user-guide/images/, I see both ImagePullSecrets and `imagePullSecrets`, maybe differing wrt the context.